pages/index.js

```diff
@@ -43,60 +43,63 @@ export default function HomePage({ trips, setTrips }) {
 
   return (
     <>
-      <StyledImageWrapper>
-        <StyledImage
-          src="/travel-log.jpg"
-          height={600}
-          width={800}
-          layout="responsive"
-          alt="Travel Notebook"
-        />
-      </StyledImageWrapper>
-      <StyledHeading1>My Travel Log</StyledHeading1>
-      <StyledHeading2>Create a new Trip</StyledHeading2>
-      <TripInputForm handleAddTrip={handleAddTrip} />
-      <StyledHeading3>Upcoming Trips</StyledHeading3>
-      <StyledSection>
-        {upcomingTrips.length === 0 ? (
-          <p>You do not have any upcoming trips yet.</p>
-        ) : (
-          upcomingTrips.map((trip) => (
-            <>
-              <StyledLink href="/explore">
-                <TripPreview key={trip.id} trip={trip} />
-              </StyledLink>
-            </>
-          ))
-        )}
-      </StyledSection>
-      <StyledHeading3>Current Trips</StyledHeading3>
-      <StyledSection>
-        {currentTrips.length === 0 ? (
-          <p>You do not have any current trips yet.</p>
-        ) : (
-          currentTrips.map((trip) => <TripPreview key={trip.id} trip={trip} />)
-        )}
-      </StyledSection>
-      <StyledHeading3>Passed Trips</StyledHeading3>
-      <StyledSection>
-        {passedTrips.length === 0 ? (
-          <p>You do not have any passed trips yet.</p>
-        ) : (
-          passedTrips.map((trip) => <TripPreview key={trip.id} trip={trip} />)
-        )}
-      </StyledSection>
+      <Scrollbox>
+        <StyledImageWrapper>
+          <StyledImage
+            src="/travel-log.jpg"
+            height={600}
+            width={800}
+            layout="responsive"
+            alt="Travel Notebook"
+          />
+        </StyledImageWrapper>
+        <StyledLogo>Travel</StyledLogo>
+        <StyledHeading1>My Travel Log</StyledHeading1>
+        <StyledHeading2>Create a new Trip</StyledHeading2>
+        <TripInputForm handleAddTrip={handleAddTrip} />
+        <StyledHeading3>Upcoming Trips</StyledHeading3>
+        <section>
+          {upcomingTrips.length === 0 ? (
+            <p>You do not have any upcoming trips yet.</p>
+          ) : (
+            upcomingTrips.map((trip) => (
+              <TripPreview key={trip.id} trip={trip} />
+            ))
+          )}
+        </section>
+        <StyledHeading3>Current Trips</StyledHeading3>
+        <section>
+          {currentTrips.length === 0 ? (
+            <p>You do not have any current trips yet.</p>
+          ) : (
+            currentTrips.map((trip) => (
+              <TripPreview key={trip.id} trip={trip} />
+            ))
+          )}
+        </section>
```

Comment:
the upcoming, current and past trip sections all have the same structure. That smells like you could build one generalized component for them! (Maybe in the future?)